app/controllers/beta/stories_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Beta::StoriesController < ApplicationController
+  before_action :set_project_and_story
+  def sort
```

Comment:
What do you think about a route `/beta/stories/:id:/position` instead of `sort`? It's for the same reason I suggested to change the JS function to `updatePosition` instead.